src/main/charts/bitbucket/README.md

```diff
@@ -18,6 +18,7 @@ Kubernetes: `>=1.17.x-0`
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| additionalConfigMapsSecrets | list | `[]` | Additional existing ConfigMaps and Secrets not managed by Helm that should be mounted into server container configMap and secret are two available types (camelCase is important!) mountPath is a destination directory in a container and key is file name name references existing ConfigMap or secret name. VolumeMount and Volumes are added with this name + index position, for example custom-config-0, keystore-2 |
```

Comment:
Hmmm, can we come up with a better name?
additionalConfigMapsAndSecrets ?
The name ephasises the source of data, while the user will likely be interested in the target, so maybe addtionalFiles ?

Reply:
yeah, not the best name indeed :) I am ok with additionalFiles. I just wanted the name to be self explanatory. I'll commit changes shortly.

Reply:
Renamed

Reply:
Typo: addtionalFiles, `i` missing.

Reply:
thanks @pbruski . Fixed :)